src/components/ExpandedText/index.tsx

```diff
@@ -0,0 +1,40 @@
+import styled from "@emotion/styled";
+import { uiColors } from "@leafygreen-ui/palette";
+import Tooltip from "@leafygreen-ui/tooltip";
+import { Disclaimer } from "@leafygreen-ui/typography";
+
+const { blue } = uiColors;
+
+interface ExpandedTextProps {
+  message: string;
+  triggerEvent?: "hover" | "click";
```

Comment:
[nit] We can type this using `TriggerEvent` which is [exported from LeafyGreen](https://github.com/mongodb/leafygreen-ui/blob/main/packages/tooltip/src/Tooltip.tsx#L46-L49).